src/main/resources/templates/db_scripts/schema.json

```diff
@@ -810,6 +810,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "effectiveLocationId",
```

Comment:
As of the beginning of Q4, RAML Module Builder based modules are now expected to provide from versions for database changes.

Please refer to [the docs](https://github.com/folio-org/raml-module-builder/blob/master/DB-schema-migration.md) for more guidance.

Reply:
Oh, it appears that is already present in the scripts section. Is there a reason this wasn't included here?

Reply:
@marcjohnson-kint, 

As far as I see this property is not applicable for indexes.

Reply:
Hi @marcjohnson-kint, I finally realized how to use this property. We have to put this `fromModuleVersion` property  on table level and update it every time we make some changes to the table. 

So added this property to the item table.

Reply:
@bohdan-suprun Thank you for investigating and working this out.

Is this described in the documentation? If not, please raise an issue for the documentation to be improved around this area.

Reply:
@marcjohnson-kint, yes this is described in the documentation: `
fromModuleVersion - this field indicates the version which the table was created/updated in. `